src/gc/env/gcenv.os.h

```diff
@@ -128,6 +128,9 @@ typedef void (*GCThreadFunction)(void* param);
 // Interface that the GC uses to invoke OS specific functionality
 class GCToOSInterface
 {
+protected:
+    static size_t s_pageSize;
```

Comment:
I think we need to add `gcenv.windows.inl` and `gcenv.unix.inl` files to allow inlining of the OS specific implementation details.

I think we will likely end up with number of these perf critical cases where inlining is important.